extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java

```diff
@@ -72,6 +74,7 @@ public interface DataSourceJdbcRuntimeConfig {
     /**
      * The max lifetime of a connection.
      */
+    @ConfigDocDefault("This feature is disabled by default.")
```

Comment:
How about "Unlimited by default"?

Reply:
By default, there is no restriction on the lifespan of a connection. _(much moon, wow)_ :)